common/os.cpp

```diff
@@ -417,4 +418,17 @@ Some auxillary functionalities might be affected. Please report this message if
 
         return escaped.str();
     }
+
+    bool is_platform_jetson()
+    {
+#ifdef __linux__
```

Comment:
1 think we can add so that no multiple usage will be done later on is keeping the answer cached somehow and returning it if called more then once..
`static`, `lazy`
I think `lazy` could be a good way here..